README.md

```diff
@@ -28,7 +35,7 @@ If you want to skip the OAuth flow you can set `BYPASS_OAUTH` to `true` in your
 
 ### Dynamics (referred to in the codebase as Achiever for historic reasons)
 
-By default the development environment expects to be able to communicate with Stem's staging Dynamics API which is used to populate the course list, and user enrolments etc. You'll need to add staging credentials to `.env` (find in lastpass: TC Achiever creds):
+By default the development environment expects to be able to communicate with Stem's staging Dynamics API which is used to populate the course list, and user enrolments etc. You'll need to add staging credentials to `.env` (find in the nonprod section of the heroku project in the terraform repo, or in RPF's lastpass: TC Achiever creds):
```

Comment:
Actually `.env.defaults` sets `ACHIEVER_USE_LOCAL_TEMPLATES=true` currently, so it's defaulting to the local templates. A side effect of this was to allow things to work out of the box (we used to have to pass a `.env` between us to get things up and running, which was error prone) as well as being handy for offline development. But I think it's a good call to make sure you have the credentials in there, and the team have an awareness when switching between these mechanisms.